README.md

```diff
@@ -0,0 +1,65 @@
+# Letter Generator for datenanfragen .de
+
+> This repository contains the source code for Letters Generation (plain text or PDF) from templates  
+
+## Installation
+
+You can install this tools via npm:  
+
+```sh
+npm i letter-generator
+```
+
+## Usage  
+
+You'll have to pass an object to the letter constructor containing those informations:  
+
+```js
+var props = {
```

Comment:
We want to encourage modern JS practices like the use of `let` and `const`. In this case, all variables can be declared as `const`.